packages/vuetify/src/components/VDatePicker/VDatePickerDateTable.ts

```diff
@@ -115,7 +115,7 @@ export default mixins(
         if (rows.length % (this.showWeek ? 8 : 7) === 0) {
```

Comment:
extract `this.showWeek ? 8 : 7` to a separate var (in line 128 as well)

otherwise looks good

Reply:
Thanks for review. Do you have a suggestion about variable name?

Reply:
Maybe rowLength or cellsInRow